OJP/OJP_FareSupport.xsd

```diff
@@ -139,45 +158,328 @@
 	<xs:annotation>
 		<xs:documentation>========================================== Booking ==========================================</xs:documentation>
 	</xs:annotation>
-	<xs:complexType name="BookingArrangementStructure">
+	<xs:simpleType name="ContactTypeEnumeration">
+		<xs:annotation>
+			<xs:documentation>Allowed values for CONTACT role. +v1.2.2</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:NMTOKEN">
+			<xs:enumeration value="any"/>
+			<xs:enumeration value="information"/>
+			<xs:enumeration value="reservations"/>
+			<xs:enumeration value="lostProperty"/>
+			<xs:enumeration value="publicRelations"/>
+			<xs:enumeration value="complaints"/>
+			<xs:enumeration value="emergency"/>
+			<xs:enumeration value="other"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="PhoneType">
+		<xs:annotation>
+			<xs:documentation>Type for an International phone number. + Country + MSIDDN e.g. +41 675601 etc.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:normalizedString"/>
+	</xs:simpleType>
+	<xs:simpleType name="EmailAddressType">
+		<xs:annotation>
+			<xs:documentation>Email address type a@b.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:normalizedString"/>
+	</xs:simpleType>
+	<xs:complexType name="ContactDetailsStructure" abstract="false">
+		<xs:annotation>
+			<xs:documentation>Type for contact details.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="ContactPerson" type="xs:string" minOccurs="0">
+				<!--NeTEx multilingualstring -->
+				<xs:annotation>
+					<xs:documentation>Name of contact person.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Email" type="EmailAddressType" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>The email address of the contact.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Phone" type="PhoneType" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Contact telephone number.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Fax" type="PhoneType" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Contact fax number.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Url" type="xs:anyURI" minOccurs="0">
+				<xs:annotation>
```

Comment:
Or should it be WebLinkStructure? I've just seen that PR #219 previously had changed anyURI to WebLinkStructure. Documentation would have to be changed in case of switching back to WebLinkStructure.